### PR TITLE
Removes 'record' and refers to the trainee

### PR DESCRIPTION
**Pull request by ralph-hawkins**
We can loose most of the uses of 'record'.

The only places I've kept it is to refer to the record as a separate entity from the trainee themselves. For example if the record is missing information (the trainee isn't missing the information).

### CODE REVIEW

**Comment by gregknight1** [comment]
That all makes sense to me @ralph-hawkins

File: app/views/new-record/check-record-apply-grouped-sections.html [hunk 1]
```diff
@@ -20,7 +20,9 @@
   {# Show a warning if the record isn’t ready to be submitted #}
   {% if not recordIsComplete %}
     {% set insetTextHtml %}
-      <p class="govuk-body">This record is not complete and cannot be submitted for TRN. If you do not have all the required information now, you can <a href="./save-as-draft" class="govuk-link">return to this draft record later</a>.</p>
+      <p class="govuk-body">
+        This trainee record is not complete and cannot be submitted for TRN. If you do not have all the required information now, you can <a href="./save-as-draft" class="govuk-link">return to this draft record later</a>.
```
Comment (edwardhorsford) [inline app/views/new-record/check-record-apply-grouped-sections.html]: Refers to both `trainee record` and `draft record` on the same line. I think we should be consistent.

File: app/views/new-record/check-record.html [hunk 2]
```diff
@@ -20,7 +20,9 @@
   {# Show a warning if the record isn’t ready to be submitted #}
   {% if not recordIsComplete %}
     {% set insetTextHtml %}
-      <p class="govuk-body">This record is not complete and cannot be submitted for TRN. If you do not have all the required information now, you can <a href="./save-as-draft" class="govuk-link">return to this draft record later</a>.</p>
+      <p class="govuk-body">
+        This trainee record is not complete and cannot be submitted for TRN. If you do not have all the required information now, you can <a href="./save-as-draft" class="govuk-link">return to this draft record later</a>.
```
Comment (edwardhorsford) [inline app/views/new-record/check-record.html]: Same as above.

File: app/views/new-record/check-record-apply-grouped-sections.html (hunk 1 above)
Comment (gregknight1) [reply app/views/new-record/check-record-apply-grouped-sections.html]: I'd omit the word 'draft' in the 2nd sentence.

Comment (ralph-hawkins) [reply app/views/new-record/check-record-apply-grouped-sections.html]: I thought remove 'record' so its consistent with the link at the bottom of the page:

localhost_3000_new-record_check-record(screenshot for design history — short) (1)

But tbh I'm not confident. We have 'trainee', 'draft' and 'record'. My preference is to go with draft in most places we have it to emphasise thats its not a 'registered trainee'.

Comment (gregknight1) [reply app/views/new-record/check-record-apply-grouped-sections.html]: Agree, @ralph-hawkins - I wasn't looking this in context, apologies.

**Review by edwardhorsford — APPROVED** [review]

**Pull request merged**